legacy-fabric-entity-events-v1/src/testmod/java/net/legacyfabric/fabric/test/entity/EntityEventsTest.java

```diff
@@ -17,36 +17,35 @@
 
 package net.legacyfabric.fabric.test.entity;
 
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
 import net.minecraft.entity.EntityType;
 
 import net.fabricmc.api.ModInitializer;
 
 import net.legacyfabric.fabric.api.entity.event.v1.ServerEntityCombatEvents;
 import net.legacyfabric.fabric.api.entity.event.v1.ServerEntityWorldChangeEvents;
 import net.legacyfabric.fabric.api.entity.event.v1.ServerPlayerEvents;
+import net.legacyfabric.fabric.api.logger.v1.Logger;
+import net.legacyfabric.fabric.impl.logger.LoggerImpl;
 
 public class EntityEventsTest implements ModInitializer {
-	private static final Logger LOGGER = LogManager.getLogger();
+	private static final Logger LOGGER = Logger.get(LoggerImpl.API, "Test", "EntityEvents");
 
 	@Override
 	public void onInitialize() {
 		ServerEntityCombatEvents.AFTER_KILLED_OTHER_ENTITY.register((entity, killedEntity) -> {
-			LOGGER.info("{} killed {}", EntityType.getEntityName(entity), EntityType.getEntityName(killedEntity));
+			LOGGER.info("%s killed %s", EntityType.getEntityName(entity), EntityType.getEntityName(killedEntity));
```

Comment:
oh god, i don't like this...
why use `%$TYPE` instead of `{}`?

Reply:
ask Player about it in Fabric discord lol.

Reply:
oh god